index.html

```diff
@@ -4,242 +4,245 @@
+                </ul>
+            </nav>
+            <nav class="hamburger">
+                <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
             </nav>
-        </nav>               
-    </header>
-    <section class="headline">
-        <div class="textbx">
-               <div>
-                  <h1 class="aboutme">
-                    I’m Sam <br> Glad to see <br> you!
-                  </h1>
-                  <p class="softdev">
-                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-                  </p>
-                  
-                </div>
-                <h2 class="connect">
-                    LET'S CONNECT
-                </h2>
-                <ul class="mediaIcons">
-                    <li class="twitter">
-                        <a href="#">
-                            <img src="images/twitter.png" alt="twitter icon">
-                         </a>
-                        </li>
-                    <li class="linkedIn">
-                        <a href="#">
-                            <img src="images/Linkedin.png" alt="LinkedIn icon">                        
-                        </a>
-                    </li>
-                    <li class="medium">
-                        <a href="#">
-                            <img src="images/medium.png" alt="Medium Icon">
-                        </a>
-                    </li>
-                    <li class="github">
-                        <a href="#">
-                           <img src="images/github.png" alt="Github Icon">
-                        </a>
-                    </li>
-                    <li class="angellist">
-                        <a href="#">
-                           <img src="images/angellist.png" alt="Angellist icon">
-                        </a>
-                    </li>
-                </ul>         
-            </div>           
+        </header>
+        <section class="headline-section">
+            <h1>I am Sam <br> Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and 
+                experience! If you like what you see and have a project y
+                ou need coded, don’t hestiate to contact me.
+            </p>
+        <div class="connect texty">
+            <h2>Lets Connect</h2>
+            <ul class="desc-social">
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
+            <li><a href="#"><i class="fab fa-medium"></i></a></li>
+            <li><a href="#"><i class="fab fa-github"></i></a></li>
+            <li><a href="#"><i class="fab fa-angellist"></i></a></li>
+            </ul>
```

Comment:
- [ ] Please make sure that each icon color is styled according to the Figma design.